unlock-app/src/middlewares/currencyConversionMiddleware.js

```diff
@@ -1,33 +1,30 @@
 // TODO convert to fetch instead of using axios when it is supported well enough
 /* eslint promise/prefer-await-to-then: 0 */
 
-import axios from 'axios'
 import { setConversionRate } from '../actions/currencyConvert'
 import configure from '../config'
+import CurrencyLookupService from '../services/currencyLookupService'
 
 const { services } = configure(global)
 const currencyPriceLookupURI = services.currencyPriceLookup
```

Comment:
I understand why you didn't change this, but perhaps it is better to put this data inside the new service? Perhaps as an exported constant, so that you don't need to use a regexp in the service. This should give us more certainty.

Reply:
Yeah, its a temporary shim to ensure that nothing breaks while I get the other side of things up and running. Although we only have provider currently, the regex in the code signals to readers of the code that this could change and please don't build around it as a given currently.

Should all be sorted by wednesday